extension/src/fileSystem/index.ts

```diff
@@ -164,6 +157,23 @@ stages:
   return appendFileSync(dvcYamlPath, pipeline)
 }
 
+export const scriptCommand = {
+  JUPYTER: 'jupyter nbconvert --to notebook --inplace --execute',
+  PYTHON: 'python'
```

Comment:
[Q] Could this be an enum?

[Q] Is there a better file to move `scriptCommand` & `getScriptCommand` to? Doesn't feel appropriate for them to be under `fileSystem`.